app/index.js

```diff
@@ -316,8 +316,16 @@ class NpmGenerator extends Generator {
      * @returns {undefined}
      */
     install() {
-        const self = this;
-        self.yarnInstall();
+        var self = this;
+        self.yarnInstall()
+            .then(() => {
+                self.spawnCommandSync('make', ['document']);
```

Comment:
What does this do? Generate API docs? 

Should we also run `make githooks` here too?